opencensus/trace_exemplar.go

```diff
@@ -19,15 +19,25 @@ package opencensus
 
 // [START monitoring_opencensus_configure_trace_exemplar]
 import (
+	"fmt"
 	"time"
 
 	googlepb "github.com/golang/protobuf/ptypes/timestamp"
 	distributionpb "google.golang.org/genproto/googleapis/api/distribution"
 	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
+	"google.golang.org/protobuf/types/known/anypb"
 )
 
-func createDataPointWithExemplar() *monitoringpb.Point {
+func createDataPointWithExemplar(projectID string) *monitoringpb.Point {
 	end := time.Now().Unix()
+	traceId := "0000000000000001"
+	spanId := "00000001"
+	spanCtx, _ := anypb.New(&monitoringpb.SpanContext{
```

Comment:
Please check the errors, can update function to return an error.

Reply:
Fixed